ibm_catalog.json

```diff
@@ -121,17 +132,19 @@
               ]
             },
             {
-              "key": "use_existing_resource_group"
-            },
-            {
-              "key": "resource_group_name"
-            },
-            {
-              "key": "prefix",
-              "required": true
+              "key": "existing_resource_group_name",
```

Comment:
for this one, we are using a display name. Please add this under the key: `"display_name": "resource_group",`

Reply:
same for security enforced variation